### ISSUE TITLE
Transparent background

### ISSUE THREAD

**Issue by lealouesdon** [MustansirZia/overlay_container#3]
Hello,

I am trying to create an overlay with content that isn't square. 
However, the OverlayContainer seems to have a gray background. 
Do you have any tips to make it transparent ?

Thanks in advance :)

**Comment by MustansirZia** [comment]
Hi @lealouesdon!

Can you kindly show a screenshot and a code sample so that I may have a better look?
:)

**Comment by lealouesdon** [comment]
This is my exemple of 3 buttons inside the overlay container
A gray background is present, and i can't seem to make it transparent...

![Screenshot from 2019-09-30 16-13-28](https://user-images.githubusercontent.com/28827040/65886897-77282100-e39d-11e9-8ec6-5bcf40e896d4.png)

**Comment by MustansirZia** [comment]
Well, could you show a code sample as well?

**Comment by lealouesdon** [comment]
yes sorry...
RoundButton is just a custom button with round edges and on a transparent background.
````
OverlayContainer(
     child: Column(
     crossAxisAlignment: CrossAxisAlignment.start,
        children: [
          Padding(
            padding: const EdgeInsets.all(8.0),
            child: RoundButton(
              child: Text(
                "1",
                textAlign: TextAlign.center,
              ),
              roundedLeft: true,
              roundedRight: true,
              callback: (){},
            ),
          ),
          Padding(
            padding: const EdgeInsets.all(8.0),
            child: RoundButton(
              child: Text(
                "2",
                textAlign: TextAlign.center,
              ),
              roundedLeft: true,
              roundedRight: true,
              callback: (){},
            ),
          ),
          Padding(
            padding: const EdgeInsets.all(8.0),
            child: RoundButton(
              child: Text(
                "3",
                textAlign: TextAlign.center,
              ),
              roundedLeft: true,
              roundedRight: true,
              callback: (){},
            ),
          )
        ],
    )),
    show: actionSelected,
    position: OverlayContainerPosition(-110.0, 120),
)
````

**Comment by MustansirZia** [comment]
For a start, can you add the buttons inside a container with a margin instead of wrapping them in  `Padding`?

**Comment by kashlo** [comment]
I had the same issue with background color
Its coming from Material widget, setting canvasColor: Colors.transparent in theme helped me

**Comment by MustansirZia** [comment]
Well, in that case, feel free to close the issue.

**Comment by rgb1380** [comment]
The issue as indicated by @kashlo is the Material widget which wraps the child widget of the OverlayEntry. @MustansirZia, please add a color property to the material widget and default to Colors.transparent.  This is a great plug-in that saves a lot of time with overlays. Will be great if it was maintained.

**Comment by MustansirZia** [comment]
I've added the `color` property to the `Material` widget and this should hopefully fix this issue. By default, it uses `Colors.transparent`.

More on this change can be found [here](https://github.com/MustansirZia/overlay_container/commit/72de588e5d51358ef5b8d6d68e84677be0562c41).

A new version `0.0.5+1` has also been released as well to pub.dev.

Really thankful to @rgb1380 and @kashlo for pointing this out. I hope this fixes this long-standing issue. And I'm really sorry it took this long and I wish I could've done this sooner. 

Cheers. :)
